Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -994,13 +996,33 @@ def test():
 
 
 ''' EXECUTION CODE '''
-client = resilient.get_client({
-    'email': USERNAME,
-    'password': PASSWORD,
-    'host': SERVER,
-    'cafile': 'true' if USE_SSL else 'false',
-    'org': ORG_NAME
-})
+
+
+def get_client():
+    if USERNAME and PASSWORD:
+        opts_dict = {
+            'email': USERNAME,
```

Comment:
lets init `opts_dict` before the condition with the common keys (host, cafile, host and org) and then just set the creds